decidim-meetings/lib/decidim/meetings/feature.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+Decidim.register_feature(:meetings) do |feature|
+  feature.engine = Decidim::Meetings::ListEngine
+  feature.admin_engine = Decidim::Meetings::AdminEngine
+
+  feature.on(:create) do |instance|
+    Decidim::Meetings::CreateMeeting.call(instance) do
+      on(:error) { raise "Can't create meeting" }
+    end
+  end
+
+  feature.on(:destroy) do |instance|
+    Decidim::Meetings::DestroyMeeting.call(instance) do
+      on(:error) { raise "Can't destroy meeting" }
+    end
+  end
+
+  feature.seeds do
+    Decidim::ParticipatoryProcess.all.each do |process|
+      next unless process.steps.any?
+
+      feature = Decidim::Feature.create!(
+        name: Decidim::Faker::Localized.sentence(2),
```

Comment:
Instead of this can we start using meaningful names? I'd add {"en" => "Meetings", "ca" => "Trobades" } etc..